Code/Scripts/CMakeLists.txt

```diff
@@ -110,17 +112,29 @@ if(SV_USE_TKCXIMAGE)
 	endif()
 endif()
 
+if (SV_USE_SYSTEM_TCL)
+	set_env_string_concat(TCL_LIBRARY "${TCL_LIBRARY_PATH}/tcl8.6" developer_script_string)
+	set_env_string_concat(TK_LIBRARY "${TK_LIBRARY_PATH}/tk8.6" developer_script_string)
+endif()
+
 if(NOT SV_USE_SYSTEM_TCL OR SV_INSTALL_SYSTEM_LIBS)
 	#Lib additions
-	set_env_string_concat(TCL_LIBRARY "${TCL_LIBRARY_PATH}/tcl8.5" developer_script_string)
-	set_env_string_concat(TK_LIBRARY "${TK_LIBRARY_PATH}/tk8.5" developer_script_string)
-	set_env_string_concat(TCL_LIBRARY "${ENV_SV_HOME}/${SV_INSTALL_TCL_LIBRARY_DIR}/tcl8.5" install_script_string)
-	set_env_string_concat(TK_LIBRARY "${ENV_SV_HOME}/${SV_INSTALL_TCL_LIBRARY_DIR}/tk8.5" install_script_string)
+	set_env_string_concat(TCL_LIBRARY "${TCL_LIBRARY_PATH}/tcl8.6" developer_script_string)
+	set_env_string_concat(TK_LIBRARY "${TK_LIBRARY_PATH}/tk8.6" developer_script_string)
+	set_env_string_concat(TCL_LIBRARY "${ENV_SV_HOME}/${SV_INSTALL_TCL_LIBRARY_DIR}/tcl8.6" install_script_string)
+	set_env_string_concat(TK_LIBRARY "${ENV_SV_HOME}/${SV_INSTALL_TCL_LIBRARY_DIR}/tk8.6" install_script_string)
 endif()
 
 #--------
 # Python
 if(SV_USE_PYTHON)
+  if (SV_USE_SYSTEM_PYTHON)
+	   append_env_string_concat(PYTHONPATH "${PYTHON_SITE_PACKAGES}" developer_script_string)
+	if (SV_USE_SYSTEM_VTK)
```

Comment:
Nope, the Windows version of python doesn't find the site-packages directory unless you make it explicit.  My guess is that the linux/mac version are just picking up the system installed version in default search paths (which don't really exist) on windows.

I'd like to keep everything as symmetric as possible, so can we add SV_USE_SYSTEM_VTK and SV_USE_VTK just like everything else?  The more things are consistent between the variables, the better.  I like the MITK terminology better (EXTERNAL...) versus "SYSTEM" since system implies part of the operating system (e.g. true with apt-get packages), but not really stuff that you compile yourself.  No need to change at this point.

We also have to think about splitting the build systems for the front-end and the flow solver.  I think it makes sense to replace "SV_FLOWSOLVER_USE_VTK" with simply SV_USE_VTK once we split the builds.

One big difference between Windows and make builds on all platforms is that Visual Studio has different configurations so all of the bin directories have subdirectories on Windows, e.g.:

Bin/RelWithDebInfo
Bin/Release
Bin/Debug
...

However, if you build with "nmake" on windows, it installs directly into "Bin" just like make does on other platforms.  This currently breaks our cmake scripts so you have to build with visual studio on Windows.

I intentionally hid most of the changes under SV_USE_SYSTEM_xxxx to avoid breaking a superbuild but anytime "xxx_SHARED_LIBRARIES" is set we should actually update the path to point to the shared libs directory.

Also note that on Windows, dll's and lib's are not the same thing.  Most cmake builds put the *.lib into the "Lib" directory, while the dll's go into the "Bin" directory (and often Bin/RelWithDebInfo on ms visual studio).